agents/craftassist/tests/test_memory.py

```diff
@@ -72,15 +73,15 @@ def test_add_mob(self):
         # add mob
         chicken = {v: k for k, v in MOBS_BY_ID.items()}["chicken"]
         mob_id, mob_type, pos, look = 42, chicken, Pos(3, 4, 5), Look(0.0, 0.0)
-        self.memory.set_mob_position(Mob(mob_id, mob_type, pos, look))
+        self.memory.nodes[MobNode.NODE_TYPE].set_mob_position(self.memory, Mob(mob_id, mob_type, pos, look))
```

Comment:
We are using `MobNode.NODE_TYPE` here and `"Triple"`, `"Chat"`, `"Program"` throughout the PR in other places, I propose that we be consistent one way or another. Let's stick with `MemoryNode.NODE_TYPE`